Task Liverpool/src/main/java/pages/SearchResultsPage.java

```diff
@@ -0,0 +1,152 @@
+    private By noResultsText = By.cssSelector(".a-headline__noResults");
+    private By firstResult;
+    List<WebElement> pricesList;
+    List<Integer> pricesIntList;
+
+    public enum SortType {
+        RELEVANCE, LOWPRICE, HIGHPRICE, RATING, VIEWED, SOLD
+    }
+
+    public SearchResultsPage(WebDriver webDriver) {
+        super(webDriver);
+    }
+
+    public int getNumberOfResultsFromUpperRightLabel(){
+        return Integer.parseInt(getElementText(numberOfResultsFromUpperRightLabel));
+    }
+
+    //Logic for clicking on the Results Page Right Arrow
+    public boolean clickNextPage(){
+        if (isElementDisplayed(rightArrow)){
+            String productId = getElementAttribute(findElement(searchResultsDisplayed), "data-prodid");
+            firstResult = By.cssSelector("li.m-product__card[data-prodid=\"" + productId + "\"]");
+            scrollToElement(rightArrow);
+            click(rightArrow);
+            return true;
+        }
+        else {
+            return false;
+        }
+    }
+
+    public int getTotalNumberOfResultsDisplayed() {
+        int totalResults = findElements(searchResultsDisplayed).size();
+
+        while (clickNextPage()){
+            totalResults += findElementsAfterACurrentElementDisappear(searchResultsDisplayed, firstResult).size();
+        }
+        return totalResults;
+    }
+
+    public void clickListViewButton(){
+        click(listViewButton);
+    }
+
+    public void clickSortProductsBy (SortType sortType) throws InterruptedException {
+
+        click(sortByDropDownList);
+        switch (sortType) {
+            case RELEVANCE:
+                click(sortByRelevance);
+                break;
+
+            case LOWPRICE:
+                click(sortByLowPrice);
+                break;
+
+            case HIGHPRICE:
+                click(sortByHighPrice);
+                break;
+
+            case RATING:
+                click(sortByRating);
+                break;
+
+            case VIEWED:
+                click(sortByViewed);
+                break;
+
+            case SOLD:
+                click(sortBySold);
+                break;
+        }
+        Thread.sleep(1000);
+    }
+
+    public void getPricesInt(){
+        pricesList = findElements(productPrice);
+        pricesIntList = new ArrayList<>();
+
+        for (WebElement prices: pricesList) {
+            pricesIntList.add(Integer.parseInt(prices.getText().substring(1).replace(",", "")));
+        }
+    }
+
+    public void addValuesToPricesIntListFromMoreResultsPages(){
+        while (clickNextPage()){
+            pricesList = findElementsAfterACurrentElementDisappear(productPrice, firstResult);
+            for (WebElement prices: pricesList) {
+                pricesIntList.add(Integer.parseInt(prices.getText().substring(1).replace(",", "")));
+            }
+        }
+    }
+
+    public boolean checkPriceIsSorted(){
```

Comment:
you are checking and comparing something... this is a test not a page functionality

Reply:
Ok, I moved this logic to the HomeTest class.